doc/glossary.rst

```diff
@@ -75,6 +75,10 @@ Glossary of Terms
       The set of Zephyr-supplied files that implement the Zephyr kernel,
       including its core services, device drivers, network stack, and so on.
 
+   power domain
+      A power domain is a collection of gates powered by the same power
+      and ground supply. Power domains on are represented by :c:struct:`device`.
```

Comment:
I don't think this description is strictly accurate. I could easily see multiple grounds being involved. The term "gates" also seems rather specific to SoC internals. Perhaps something along the lines of
```
is a collection of devices for which power is applied and removed collectively in a single action. 
```

Reply:
I liked it.